app/gulp/js/statistics.js

```diff
@@ -0,0 +1,133 @@
+
+    const layers = stack(data);
+
+    xScale.domain(data.map(d => d.label));
+    yScale.domain([0, d3.max(layers[layers.length - 1], d => d[0] + d[1])]).nice();
+
+    const layer = svg.selectAll('.layer')
+      .data(layers)
+      .enter().append('g')
+      .attr('class', 'layer')
+      .style('fill', (d, i) => color(i));
+
+    layer.selectAll('rect')
+        .data(d => d)
+      .enter().append('rect')
+        .attr('x', d => xScale(d.data.label))
+        .attr('y', d => yScale(d[1]))
+        .attr('width', xScale.bandwidth())
+        .attr('height', d => (yScale(d[0]) - yScale(d[1])))
+        .on('mousemove', function (d, i) {
+          d3.select(this).attr('opacity', 0.5);
+          div.transition()
+            .duration(20)
+            .style('opacity', 0.9);
+          console.log(d);
+          if (d.data.correct === d[1] - d[0]) {
+            div.html(`Q: ${d.data.label}<br/>Correct:${
+                d[1] - d[0]}`);
+          } else {
+            div.html(`Q: ${d.data.label}<br/>Incorrect:${
+                d[1] - d[0]}`);
+          }
+          div.style('left', `${d3.event.pageX + 15}px`)
+            .style('top', `${d3.event.pageY - 20}px`);
+        })
+        .on('mouseout', function (d, i) {
+          d3.select(this).attr('opacity', 1);
+          div.transition()
+            .duration(20)
+            .style('opacity', 0);
+        });
+
+    svg.append('g')
+        .attr('class', 'axis axis--x')
+        .attr('transform', `translate(0,${height + 5})`)
+        .call(xAxis);
+
+    svg.append('g')
+        .attr('class', 'axis axis--y')
+        .attr('transform', 'translate(0,0)')
+        .call(yAxis);
+
+    svg.append('text')
+        .attr('class', 'y label')
+        .attr('text-anchor', 'end')
+        .attr('y', 6)
+        .attr('dy', '.75em')
+        .attr('transform', 'rotate(-90)')
+        .text('Total Answers');
+
+    svg.append('text')
+        .attr('class', 'x label')
+        .attr('text-anchor', 'end')
+        .attr('x', width)
+        .attr('y', height - 6)
+        .text('Questions');
+  }
+
+  function showStats() {
+    const results = $('.stats_class').data('results');
+    const q = $('.stats_class').data('questions');
+    const questions = {};
+    for (let i = 0; i < q.length; i += 1) {
+      questions[q[i].id] = q[i];
+    }
+    if (!results) {
+      return;
+    }
+    const qstats = {};
+    for (let i = 0; i < results.length; i += 1) {
+      if (results[i].question_id) {
+        if (!qstats[results[i].question_id]) {
+          qstats[results[i].question_id] = { label: questions[results[i].question_id].content,
+            correct: 0,
+            incorrect: 0 };
+        }
+        if (results[i].correct) {
+          qstats[results[i].question_id].correct += 1;
+        } else {
+          qstats[results[i].question_id].incorrect += 1;
+        }
+      }
+    }
+    const data = $.map(qstats, (v, i) => [v]);
+    draw(data);
+  }
+  showStats();
+});
```

Comment:
Remove all the unused variables and `console.log`s that the linter is complaining about (they should be unnecessary anyway).

I'm also seeing a TypeError.
```
Uncaught TypeError: Cannot read property 'length' of undefined
    at statistics.js:108
    at statistics.js:129
    at HTMLDocument.<anonymous> (jquery.turbolinks.self-176b981….js?body=1:30)
    at HTMLDocument.dispatch (jquery.self-bd7ddd3….js?body=1:5227)
    at HTMLDocument.elemData.handle (jquery.self-bd7ddd3….js?body=1:4879)
    at Object.trigger (jquery.self-bd7ddd3….js?body=1:5131)
    at HTMLDocument.<anonymous> (jquery.self-bd7ddd3….js?body=1:5861)
    at Function.each (jquery.self-bd7ddd3….js?body=1:371)
    at jQuery.fn.init.each (jquery.self-bd7ddd3….js?body=1:138)
    at jQuery.fn.init.trigger (jquery.self-bd7ddd3….js?body=1:5860)
    at HTMLDocument.onLoad (jquery.turbolinks.self-176b981….js?body=1:35)
    at fire (jquery.self-bd7ddd3….js?body=1:3233)
    at Object.fireWith [as resolveWith] (jquery.self-bd7ddd3….js?body=1:3363)
    at Function.ready (jquery.self-bd7ddd3….js?body=1:3583)
    at HTMLDocument.completed (jquery.self-bd7ddd3….js?body=1:3618)
```